fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1120,9 +1120,6 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static long tablet_sched_colocate_be_down_tolerate_time_s = 12L * 3600L;
 
-    @ConfField(aliases = {"tablet_balancer_strategy"})
```

Comment:
also  i think we need to adjust default value of `tablet_sched_slot_num_per_path` to 4, since balance and repair use the same slot

Reply:
and don't forget to update the documentation